### ISSUE TITLE
py-nio:  Port health display fails to update from "failed" to "successful"

### ISSUE THREAD

**Issue by Dave-Allured** [macports/macports-webapp#361]
Port **py-nio** was recently rebuilt by manual triggers, after dependency **py27-nio** was fixed.  However, the new **py-nio** builds are not syncing to the ports website.  Build status continues to show only old broken builds, and "failed install-dependencies" for 10 out of 11 builders that should now be "successful".  Therefore the port health is seriously misstated.

On previous advice, I have avoided trying to force by rev-bump.  Is there a bug in the port sync machinery, or am I missing something?

https://ports.macports.org/port/py-nio/builds/
https://ports.macports.org/port/py27-nio/builds/

[Updated description to be more specific to port health display.]

**Comment by Dave-Allured** [comment]
Now I see it.  This is related to https://github.com/macports/macports-webapp/issues/343.

**Comment by jmroot** [comment]
> This is related to #343.

I don't think so. There really were no successful builds of py-nio on anything other than Ventura x86_64, because it is a stub port and thus the built archive can be used on any OS version, so the other builders all skipped it. So this is more like #135 with the added complication of `platforms any`.

**Comment by Dave-Allured** [comment]
@jmroot, I see.  It looks like rev bump is the only way for a developer to fix displayed port health after a failed dependency is fixed.  Agreed?

**Comment by jmroot** [comment]
No, a rev bump will not help anything here.
